### ISSUE TITLE
Error 401 on GET https://api-v2launch.trakt.tv/users/me

### ISSUE THREAD

**Issue by zedug** [tegon/traktflix#25]
Nothing is happening when I try to login to trakt.tv apart from the little red circle going around. Console gives the error message mentioned in the title.
Not sure if it's related to the plugin, my browser, or trakt ?
Any pointers welcome...

using Chrome Canary build 50.0.2642.0 (Official Build) canary (64-bit)
traktflix 1.0.5.5

thanks

**Comment by tegon** [comment]
When you open Netflix, the extension try to get your info so the 401 is normal if you're unlogged.
When you click login, the trakt popup appears to you?

**Comment by zedug** [comment]
You mean when I click login in Netflix ? Nope. I login Netflix, go to the view history page, click on traktflix, the popup opens, I click on "login with trakt.tv", and it gets stuck on the red circle.

**Comment by tegon** [comment]
Can you run this on your console (with Netflix open) and past the results here?
```js
chrome.storage.local.get(function(data) { console.log(data); })
```
Important: you have to switch from top frame to traktflix on the console
![screen shot 2016-02-10 at 8 59 33 am](https://cloud.githubusercontent.com/assets/3727827/12945475/b7c4111e-cfd4-11e5-81a4-fbaf547325d4.png)

**Comment by zedug** [comment]
here's the result (note that I'm not using any extension that would block data)

![clipboard-2](https://cloud.githubusercontent.com/assets/9880558/12985797/a62a26c0-d0f4-11e5-8d6d-1d6f944a86c4.jpg)

**Comment by tegon** [comment]
This is really weird, so you never get to see this screen?
![screen shot 2016-02-18 at 9 11 27 am](https://cloud.githubusercontent.com/assets/3727827/13141672/b3603840-d61f-11e5-8ade-bfeb0ca6e077.png)
The results you posted above show that you don't have a access token in your storage yet, so when you click login that screen should popup for you to fill with Trakt.tv credentials.
I'll see if something has changed on canary. BTW, do you have a stable chrome version to test?

**Comment by zedug** [comment]
So I tried again yesterday and the behavior has changed.

With Canary I do now have access to that screen, however it's all in "read only'", I can't edit the username and password fields.

With Chrome (which I normally don't use because I have the "aw snap" issue all the time so forced to launch it with --no-sandbox so I don't like it), I have a constant turning red circle -- back to square one..

_//EDIT
I retried this morning before going to work, and after rebooting, and Chrome just worked. I had the trakt credential screens, i authorized the app, and did the history sync.

...so I don't really what changed (still on 1.0.5.5) but thanks for your help and perseverance !_
